Gemfile

```diff
@@ -158,6 +158,8 @@ group :development, :test do
   # Rails add-on for static analysis.
   gem 'rubocop-performance'
   gem "rubocop-rails", "~> 2.25.1"
+  # More concise test ("should") matchers
+  gem "shoulda-matchers", "~> 6.2"
```

Comment:
Moved this gem because otherwise `rails g model` fails because `shoulda-matcher` is needed for factory bot to create the spec factory files, so we need `shoulda-matcher` in the dev environment as well.